network/p2p/test_utils.go

```diff
@@ -131,14 +130,15 @@ func (ln *LocalNet) NewTestP2pNetwork(ctx context.Context, keys testing.NodeKeys
 	if err != nil {
 		return nil, err
 	}
-	cfg := NewNetConfig(keys.NetKey, format.OperatorID([]byte(operatorPubkey)), ln.Bootnode, testing.RandomTCPPort(12001, 12999), ln.udpRand.Next(13001, 13999), maxPeers)
+	cfg := NewNetConfig(keys, format.OperatorID([]byte(operatorPubkey)), ln.Bootnode, testing.RandomTCPPort(12001, 12999), ln.udpRand.Next(13001, 13999), maxPeers)
```

Comment:
this line is confusing, again we need to get it clear if its operator ID or pubkey.
```go
format.OperatorID([]byte(operatorPubkey)
```

Reply:
this like the other comment is legacy code that hasn't been changed in this pr.